rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -639,6 +641,17 @@ public byte[] getStateForBtcReleaseClient(Object[] args) throws VMException {
         }
     }
 
+    public byte[] getStateForSvpClient(Object[] args) throws VMException {
+        logger.trace("getStateForSvpClient");
+
+        try {
+            return bridgeSupport.getStateForSvpClient();
```

Comment:
Does this throw any exception?

Reply:
This method does not throw any checked exception. But if it throws an unchecked exception, for whatever reason, we should catch it and throw a `VMException`.